assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -240,12 +233,6 @@ const baseSelectors = {
 		}
 
 		if ( answerBasedMetrics.length ) {
-			if ( ! isFeatureEnabled( 'conversionReporting' ) ) {
-				return answerBasedMetrics.filter( ( slug ) => {
-					return keyMetricsGA4WidgetsNonACR.includes( slug );
-				} );
-			}
-
```

Comment:
Same here, will restore it in my current PR

Reply:
@zutigrm  In `getAnswerBasedMetrics`, we already check for the feature flag and adjust the values of what is returned. So no point in doing this again.